charts/victoria-logs-multilevel/_index.md.gotmpl

```diff
@@ -51,17 +51,39 @@ The chart requires setting the list of VictoriaLogs addresses to query in order
 according to [multi-level VictoriaLogs setup](https://docs.victoriametrics.com/victorialogs/cluster/#multi-level-cluster-setup).
 The list of these addresses must be set via `.Values.storageNodes` option.
 
-Let's deploy multilevel VictoriaLogs chart that queries three VictoriaLogs clusters deployed via [victoria-logs-cluster Helm chart](https://docs.victoriametrics.com/helm/victoria-logs-cluster/):
-- `vl-cluster-1:8427`
-- `vl-cluster-2:8427`
-- `vl-cluster-3:8427`
+To demonstrate how multi-level VictoriaLogs chart work, let's deploy three [VictoriaLogs single-node instances](https://docs.victoriametrics.com/helm/victoria-logs-single/#install-victoria-logs-single-chart) with different retention times and [`victoria-logs-collector`](https://docs.victoriametrics.com/helm/victoria-logs-collector/) chart:
+
+```
+export NAMESPACE=logging
+export VL1_RETENTION=1d
+export VL2_RETENTION=7d
+export VL3_RETENTION=30d
+export VL1_PVC_SIZE=10Gi
+export VL2_PVC_SIZE=20Gi
+export VL3_PVC_SIZE=30Gi
+
+helm install vl-cluster-1 vm/victoria-logs-single --namespace $NAMESPACE --create-namespace --wait \
```

Comment:
The `vl-cluster-*` name is a bit misleading, since there is no cluster there. Let's give it less misleading name without the `cluster` word. For example, `vl-*`.